std/container/rbtree.d

```diff
@@ -1669,6 +1670,12 @@ assert(equal(rbt[], [5]));
         }
     }
 
+    void toString(scope void delegate(const(char)[]) sink, FormatSpec!char fmt) const {
```

Comment:
Doing a search through Phobos I see examples of both documentation and no documentation, so your call.